ref/general/data-persistence-jpa.adoc

```diff
@@ -0,0 +1,49 @@
+The JPA specification defines an EntityManager API, which can persist, update, retrieve, or remove objects from a database. The EntityManager API is a resource manager that maintains the active collection of entity instances that are used by the application.  Each EntityManager instance is associated with a persistence context. A persistence context defines the scope under which particular entity instances are created, persisted, and removed by the EntityManager API.
+
+The EntityManager tracks all entities within a persistence context for changes and updates and flushes these changes to the persistent storage database. Once a persistence context is closed, all managed entities become detached from the persistence context and its associated EntityManager. After an entity is detached from a persistence context, it is longer managed by an EntityManager, and state changes are not synchronized with the database.
+
+=== Persistence units and the EntityManagerFactory interface
+An application in a container can obtain the EntityManager through injection into the application or by looking it up in the Java component namespace. If the application manages its own persistence, the EntityManager is obtained from the EntityManagerFactory interface. This interface draws on declarative metadata that describes the relationship of entity class objects to a relational database. This data is known as a persistence unit, and the EntityManagerFactory interface uses it to create a persistence context that that can be accessed through the EntityManager instance.
+
+=== JPA and JDBC
+
+The Java Database Connectivity API (JDBC) handles the actual communication between an application and the tables in a relational database by managing SQL queries and commands. JPA provides a level of abstraction between the application and JDBC layers, allowing developers to manage persistence with Java annotations or XML, without having to worry about the details of SQL functions. Although managing persistence at the JDBC level provides more flexibility and fine-grained control  over how data is persisted, it is much more complex to configure and maintain. Whether you manage persistence through JPA or directly with JDBC depends on the needs of your application. For more information about JDBC, see [link pending].
+
+== JPA configuration with Open Liberty
+
+JPA for Open Liberty is supported by the Java Persistence API feature, version 2.1 and later. This feature includes Java Persistence API 2.1 and later specification interfaces and container-managed JPA integration and also includes EclipseLink as the default JPA provider implementation. This feature also enables the the Java Persistence API Container feature, which supports configuring JPA provider implementations other than the default. For more information, see link:/docs/ref/feature/#jpa.html[the Java Persistence API feature] and link:/docs/ref/feature/#jpaContainer.html[the Java Persistence API Container feature].
```

Comment:
This sentence reads a little bit funny to me I think because of the multiple instances of "and":

>This feature includes Java Persistence API 2.1 and later specification interfaces and container-managed JPA integration and also includes EclipseLink as the default JPA provider implementation.